core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -164,6 +164,19 @@ public class RestConfig extends AbstractConfig {
       + " be used to specify additional tags during deployment like data center, instance "
       + "details, etc.";
   protected static final String METRICS_TAGS_DEFAULT = "";
+  public static final String METRICS_ENABLE_LATENCY_SLO_SLA_CONFIG =
+      "metrics.enable.latency.slo.sla";
```

Comment:
We would typically use a name ending in `enable` for a config. Such as `metrics.latency.slo.sla.enable`.